rpc/client.go

```diff
@@ -16,18 +26,34 @@ import (
 )
 
 type UploadReportOptions struct {
-	Report          *poplar.Report
-	ClientConn      *grpc.ClientConn
-	SerializeUpload bool
-	DryRun          bool
+	Report             *poplar.Report
+	ClientConn         *grpc.ClientConn
+	SerializeUpload    bool
+	AWSSecretKey       string
+	AWSAccessKey       string
+	ResultsHandlerHost string
+	ResultType         string
+	DryRun             bool
+}
+
+type SignedUrl struct {
+	Signed_Url      string
+	Expiration_secs int
 }
 
 func UploadReport(ctx context.Context, opts UploadReportOptions) error {
 	if err := opts.convertAndUploadArtifacts(ctx); err != nil {
 		return errors.Wrap(err, "uploading tests for report")
 	}
-	return errors.Wrap(uploadTests(ctx, gopb.NewCedarPerformanceMetricsClient(opts.ClientConn), opts.Report, opts.Report.Tests, opts.DryRun),
-		"uploading tests for report")
+	err := errors.Wrap(uploadTests(ctx, gopb.NewCedarPerformanceMetricsClient(opts.ClientConn), opts.Report, opts.Report.Tests, opts.DryRun), "uploading tests for report")
+	if err != nil {
+		return err
+	}
+	err = errors.Wrap(uploadResultsToPSS(opts.Report, opts.AWSSecretKey, opts.AWSAccessKey, opts.ResultsHandlerHost, opts.ResultType, opts.DryRun), "uploading results to PSS")
+	if err != nil {
+		return err
+	}
```

Comment:
This may be paranoid, but I actually think we shouldn't error on the perf summarizer service submission just yet. Just in case something goes wrong during dev, I'd rather keep this "invisible". During the later followup ticket when we remove submission to cedar, we can make failure to submit to perf-summarizer-service an error.

Reply:
(Although logging on error seems reasonable!)

Reply:
+1